src/components/site/SubPageTabs.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const SubPageTabs = props => {
+  const { text, id, isExpanded = true } = props;
+  return (
+    <div className="dg_subpage-tabs-container">
+      <div className="container">
```

Comment:
I think we were going to try and use sass with @extend to try and accomplish this, and to remove this particular div?

Reply:
Updated to remove the `container` class. Set `dg_subpage-tabs `to handle the container width at the proper breakpoint and added the proper padding and margins to match.